python/dgl/distributed/rpc_server.py

```diff
@@ -70,10 +70,15 @@ def start_server(server_id, ip_config, num_clients, \
         for client_id, _ in client_namebook.items():
             register_res = rpc.ClientRegisterResponse(client_id)
             rpc.send_response(client_id, register_res)
-    server_state = get_server_state()
+    server_state = ServerState(kv_server)
```

Comment:
i don't think you can create server state here. what should `get_server_state` returns?

Reply:
I just put the server_state as an argument passed to start_server.